src/content/en/2020/seo.md

```diff
@@ -637,7 +636,7 @@ Good | 15.44% | 8.39%
 Average | 25.49% | 20.19%
 Poor | 59.06% | 71.42%
 
-<figcaption>{{ figure_link(caption="Good, Average and Poor ratios of Lighthouse v5 versus v6", sheets_gid="692150551", sql_file="TODO.sql") }}</figcaption>
+<figcaption>{{ figure_link(caption="Good, Average and Poor ratios of Lighthouse v5 versus v6", sheets_gid="692150551", sql_file="TODO..sql") }}</figcaption>
```

Comment:
@Tiggerito can you suggest the correct SQL file for this figure?

Reply:
I think that came from the performance chapter @fellowhuman1101 ?

Reply:
Ah interesting, so if the query itself doesn't live in the SEO directory then this value wouldn't be very useful and we can omit it.

Reply:
It’s my intention to handle a full path here, or even relative (`../performance/query.sql`) in much the same way as we do for images. So would still be good to get the SQL from performance directory.

Reply:
It will work with relative but not absolute. Will need to test to be 100% sure but pretty confident. Plan on looking at this, this weekend when I’ll add absolute support too. Will try to finish this out for launch - saw your updates but not had a chance to look at them yet.

Reply:
> I think that came from the performance chapter @fellowhuman1101 ?

Correct, this came from the performance chapter.

Reply:
This figure seems to have been removed, so no issue anymore.